<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainBMSWindow</class>
 <widget class="QMainWindow" name="MainBMSWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainBMSWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>271</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="title">
     <string>CAN通讯</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>230</width>
       <height>24</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="minimumSize">
         <size>
          <width>40</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>40</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>波特率</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBox_bps">
        <property name="minimumSize">
         <size>
          <width>70</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>70</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <item>
         <property name="text">
          <string>250Kbps</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_connect">
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>连接</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_discon">
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>断开</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>170</y>
      <width>271</width>
      <height>51</height>
     </rect>
    </property>
    <property name="title">
     <string>辨识阶段</string>
    </property>
    <zorder>groupBox_9</zorder>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>30</y>
      <width>561</width>
      <height>71</height>
     </rect>
    </property>
    <property name="title">
     <string>充电参数配置阶段</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>110</y>
      <width>561</width>
      <height>121</height>
     </rect>
    </property>
    <property name="title">
     <string>充电阶段</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_5">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>260</y>
      <width>371</width>
      <height>101</height>
     </rect>
    </property>
    <property name="title">
     <string>充电结束阶段</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_6">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>240</y>
      <width>651</width>
      <height>201</height>
     </rect>
    </property>
    <property name="title">
     <string>发送报文</string>
    </property>
    <widget class="QTableView" name="tableViewSend">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>631</width>
       <height>171</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Ubuntu Mono</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_7">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>460</y>
      <width>651</width>
      <height>181</height>
     </rect>
    </property>
    <property name="title">
     <string>接收报文</string>
    </property>
    <widget class="QTableView" name="tableViewReceive">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>631</width>
       <height>151</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Ubuntu Mono</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_8">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>390</y>
      <width>371</width>
      <height>111</height>
     </rect>
    </property>
    <property name="title">
     <string>警示信息</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_9">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>100</y>
      <width>271</width>
      <height>51</height>
     </rect>
    </property>
    <property name="title">
     <string>握手阶段</string>
    </property>
   </widget>
   <widget class="QToolBox" name="toolBox">
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>0</y>
      <width>151</width>
      <height>211</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="page">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>151</width>
       <height>159</height>
      </rect>
     </property>
     <attribute name="label">
      <string>握手阶段</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="page_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>151</width>
       <height>159</height>
      </rect>
     </property>
     <attribute name="label">
      <string>辨识阶段</string>
     </attribute>
    </widget>
   </widget>
   <widget class="QCheckBox" name="checkBox">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>80</y>
      <width>101</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>物理连接完成</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
